core/statistics/softwareVersion/softwareVersion_test.go

```diff
@@ -0,0 +1,37 @@
+package softwareVersion
+
+import (
+	"testing"
+
+	"github.com/ElrondNetwork/elrond-go/consensus/mock"
+	"github.com/ElrondNetwork/elrond-go/core/appStatusPolling"
+	"github.com/stretchr/testify/assert"
+)
+
+func TestNewSoftwareVersionChecker_NilStatusHandlerShouldErr(t *testing.T) {
+	t.Parallel()
+
+	softwareChecker, err := NewSoftwareVersionChecker(nil)
+
+	assert.Nil(t, softwareChecker)
+	assert.Equal(t, appStatusPolling.ErrNilAppStatusHandler, err)
```

Comment:
also expect here the error you will declare in the same package

Reply:
done.